libs/base/form/src/renderers/default-form.renderer.ts

```diff
@@ -5,13 +5,25 @@ import { classMap, ClassMapDirective } from 'lit/directives/class-map.js';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import { repeat } from 'lit/directives/repeat.js';
 import { when } from 'lit/directives/when.js';
-import { FormFieldDefinition, FormFieldType, FormValues } from '../models';
+import {
+  FieldValidationPattern,
+  FormFieldDefinition,
+  FormFieldType,
+  FormValues,
+} from '../models';
 import { FormRenderer } from './form.renderer';
 import { FormFieldRenderer } from './renderer';
 
 export class DefaultFormRenderer implements FormRenderer {
   constructor(protected injector = inject(INJECTOR)) {}
 
+  fieldValidationPattern(field: FormFieldDefinition): FieldValidationPattern {
```

Comment:
Also do we want to expose this method as public API?

Reply:
Yeah, I believe so. For cases when we are building custom fields like country select in address form component.